### ISSUE TITLE
Wotlk Lua error on log in

### ISSUE THREAD

**Issue by Speaker1264** [Jaliborc/Bagnon#1872]
**Which software were you running?**
 - Addon version name: Bagnon-10.2.18
 - Client used: WotLK Classic

**Have you read the changelog?**
Yes

**Please describe the bug.**
Just a simple lua error. Not sure what it's affecting

**Please describe how to reproduce it.**
1. Log into the game
2. Be greeted with a lua error as soon as you load in

**Error Logs are Important!**
10x [string "BagBrother/libs/TaintLess/..."]:77: hooksecurefunc(): GetAttribute is not a function
[string "=[C]"]: in function `hooksecurefunc'
[string "BagBrother/libs/TaintLess/TaintLess.xml:<Scripts>"]:77: in main chunk

Locals:

**Comment by Jaliborc** [comment]
Yeah, taintless sometimes does that, but reduces taint in combat, which is more important I'd argue
